src/callbacks_step/analysis.jl

```diff
@@ -195,6 +201,14 @@ function (analysis_callback::AnalysisCallback)(integrator)
   @unpack dt, t = integrator
   iter = integrator.destats.naccept
 
+  # Record performance measurements and compute walltime/coretime PID
+  runtime_since_last_analysis = 1.0e-9 * (time_ns() - analysis_callback.start_time_last_analysis)
+  ncalls_rhs_since_last_analysis = (semi.performance_counter.ncalls_since_readout -
+                                    analysis_callback.ncalls_rhs_last_analysis)
```

Comment:
The subtraction should not be necessary if we keep the readout of the performance counter.

Reply:
Sorry, I don't follow. What do you mean by "keeping the readout"?

Reply:
`take!(semi.performance_counter)` below

Reply:
Yes, but `take!` directly computes the average of the time, right? It does not allow me to extract only the number of `ncalls`. Or do you mean that it should not be necessary to store `analysis_callback.ncalls_rhs_last_analysis` since by definition it will always be zero?

Reply:
While we are talking about it: Would you feel like the way it is implemented right now (maybe adding an accessor `ncalls(performance_counter)` instead of relying on implementation details and reaching directly into the struct) is "non-hacky" enough or that I should change how I determine the number of `rhs!` calls for the PID computations?

Reply:
> Yes, but `take!` directly computes the average of the time, right? It does not allow me to extract only the number of `ncalls`. Or do you mean that it should not be necessary to store `analysis_callback.ncalls_rhs_last_analysis` since by definition it will always be zero?

Yes to both :+1:

Reply:
> While we are talking about it: Would you feel like the way it is implemented right now (maybe adding an accessor `ncalls(performance_counter)` instead of relying on implementation details and reaching directly into the struct) is "non-hacky" enough or that I should change how I determine the number of `rhs!` calls for the PID computations?

Adding an `ncalls` accessor would definitely be better to make this work nicely with the way how we measure the parabolic stuff.

Reply:
Done in f815200d.